src/vue-sl-model.js

```diff
@@ -4,13 +4,14 @@ export default {
     const wm = new WeakMap();
     app.directive("sl-model", {
       beforeMount(el, binding, _vnode) {
+        const eventName = el.tagName.includes("SELECT") ? "sl-change" : "input";
```

Comment:
Nice catch. Can we make this more explicit? Currently, Shoelace tag names are "hard coded" and can't be changed, so checking for `SL-SELECT` would be preferred.

Reply:
True. It's been done. :)